File: CC_api/Models/Participant.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace CC_api.Models
{
  public class Participant
  {
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public int participantid { get; set; }
    public int conversationid { get; set; }
    public int user_id { get; set; }
    public string first_name { get; set; }
    public string last_name { get; set; }
    public int company_id { get; set; }
    public string company_name { get; set; }
  }
}

File: cc-ui/src/app/DTO/Participant.ts
export interface participant{
    participantId?:number,
    conversationId:number,
    userId:number,
    companyId:number,
    first_name: string,
    last_name:string,
    company_name:string,
    AdscompanyId:number;
}
export interface Candidate{
    user_id:number,
    company_id:number,
    first_name:string,
    last_name:string,
    designation:string
    selected:boolean
    company_name: string
    AdscompanyId:number;
}